server/modules/user/controllers/index.js

```diff
@@ -30,20 +32,14 @@ router.get(
   ]),
   getUsers,
 );
-router.patch(
-  '/',
-  csrfProtection,
-  authenticate(),
-  authorize([
-    /* super admin */
-  ]),
-  updateUser,
-);
+router.get('/admin-users', getAdminUsers);
+router.patch('/update-user-role', csrfProtection, authenticate(), updateUser);
```

Comment:
add authentication and authorization for both routes

Reply:
done